src/Vision/VNFaceLandmarkRegion2D.cs

```diff
@@ -0,0 +1,40 @@
+﻿//
+// VNFaceLandmarkRegion2D.cs
+//
+// Authors:
+//	Alex Soto  <alexsoto@microsoft.com>
+//
+// Copyright 2017 Xamarin Inc. All rights reserved.
+//
+
+#if XAMCORE_2_0
+
+using System;
+using Vector2 = global::OpenTK.Vector2;
+
+namespace XamCore.Vision {
+	public partial class VNFaceLandmarkRegion2D {
+
+		public Vector2 this [nuint index] {
+			get { return GetPoint (index); }
+		}
+
+		public Vector2 [] Points {
+			get {
+				var ret = _GetPoints ();
+				if (ret == IntPtr.Zero)
+					return null;
```

Comment:
I'm mimicking what the native API is giving us, if we get nothing from the unmanaged API we return nothing if we get an empty NSArray it will return an empty `Vector2` array :)